include/types.h

```diff
@@ -685,6 +686,7 @@ typedef enum user_options_map
 {
   IDX_ADVICE_DISABLE            = 0xff00,
   IDX_ATTACK_MODE               = 'a',
+  IDX_AUTODETECT_ONLY           = 0xff4d,
```

Comment:
Is a bit annoying, but we try to keep the numbering synchronous with the order of the macro. That means that  from the point where you insert a new macro you need to update all subsequent macros +1.